manifests/service.pp

```diff
@@ -50,11 +56,20 @@
 
 ### Manage actions
 
-  service { $repose::params::service:
-    ensure     => $service_ensure,
-    enable     => $enable,
-    hasstatus  => $repose::params::service_hasstatus,
-    hasrestart => $repose::params::service_hasrestart,
-  }
+  if $container == 'valve' {
```

Comment:
It's worth noting that with repose 9 the only deployment is valve. I understand how the flag is being used here, but i thought i'd make sure it was known.

Reply:
This is to handle backward compatibility. Further down on L#66 certain conventions have to be set because even tho repose9 is valve only, the service name was changed.